commands/rm.go

```diff
@@ -62,7 +81,13 @@ func rmCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 		Args:  cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			options.builder = rootOpts.builder
+			if !options.allInactive && options.force {
+				return errors.New("cannot specify --force without --all-inactive")
```

Comment:
nit: Not sure if this error is needed as it doesn't really matter for the user's intent. I guess the current behavior is that single `rm` is "always forced". Maybe that would change in the future.

Reply:
> nit: Not sure if this error is needed as it doesn't really matter for the user's intent.

to be fair I was not sure about that one too. will remove this cond.

> I guess the current behavior is that single `rm` is "always forced".

yes that's the current behavior.